### ISSUE TITLE
Android app crashes on split screen view on Samsung tablet

### ISSUE THREAD

**Issue by CmplxAdSys** [bitwarden/android#2327]
### Steps To Reproduce

1. Open up browser (I use Brave) on Samsung tablet in full screen view
2. Open bitwarden so that browser is on one side of the screen and bitwarden is on the other
3. Launch a website in bitwarden. The website should open in a tab on the browser on the other side of the screen
4. Launch another website from bitwarden as above. 
5. Repeat 2 or more times, which will crash the bitwarden app

### Expected Result

The bitwarden app does not crash when in split screen view and launching websites from the app

### Actual Result

Bitwarden crashes

### Additional Context

On a Galaxy tab S6

### Operating System

Android

### Operating System Version

12

### Device

Samsung Galaxy Tab S6

### Build Version

Version: 2023.1.0 (5786)

### Beta

- [ ] Using a pre-release version of the application.

**Comment by CmplxAdSys** [comment]
I attempted to reproduce but also couldn't. However, I still had bitwarden crash in split screen mode in two other ways, but both don't happen all the time. They happen after a little use of bitwarden being in split screen. 

1) When it was reduced to quarter-screen. Here's how you do that:
- Open any app full screen in landscape view on the tablet
- Put bitwarden on the right side. This yields a split screen.
- Swip the side panel to reveal side panel apps. Tap any app. This opens up the new app in the bottom right of the screen. Sometimes, bitwarden properly resizes to the top right quarter of the screen. Sometimes it crashes.
2) Tapping sites in split=screen view
-Open bitwarden in split-screen view with any app on the left and bitwarden on the right. 
- Tap logins
- Tap any site and (sometimes) bitwarden crashes

There are both hard to catch bugs! I just spent two minutes trying to reproduce the above crashes, and I couldn't get it to reliably happen. But bitwarden almost always crashes at least once while using it in split-screen view for me. It's just hard to reproduce the exact steps that lead to its crash. It always happens when interacting somehow with bitwarden in split screen view, but it doesn't always happen the same way and following the same steps.
